.packit.yaml

```diff
@@ -0,0 +1,12 @@
+specfile_path: ci/rubygem-abrt.spec
+jobs:
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-all
+actions:
+  create-archive:
+  -  bash -c "gem build *.gemspec; mv -f *.gem ci/; ls ci/*.gem"
```

Comment:
Ugh, I had old .gem files lying around and now they are moved to `ci/` folder, that is not nice

Reply:
If just RubyGems were not so broken :exploding_head: 

https://github.com/rubygems/rubygems/issues/3953

Reply:
Oh, I didn't think of that. But I'm glad I didn't have anything like `rm *.gem` there in the beginning.

Unfortunately, I couldn't think of any other (generic) way to do it. Ideally, there wouldn't be any of moving, or changing directories, but the `.gem` file needs to be in `ci/` folder for packit to find it (not sure whether that's a bug; I noted that in some issue, but maybe it's in `it works, don't touch it` state).

Creating the gem using `-C` would definitely help. Thanks for starting that thread, and creating PR!